index.html

```diff
@@ -1,64 +1,52 @@
-      name="description"
-      content="Um dicionário tech para pessoas que querem aprender mais sobre termos técnicos dentro da tecnologia!"
-    />
+<head>
+  <!-- Meta settings -->
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="description"
+    content="Um dicionário tech para pessoas que querem aprender mais sobre termos técnicos dentro da tecnologia!" />
 
-    <!-- Favicon -->
-    <link
-      rel="shortcut icon"
-      href="/assets/img/brand/symbol/colored.ico"
-      type="image/x-icon"
-    />
+  <!-- Favicon -->
+  <link rel="shortcut icon" href="/assets/img/brand/symbol/colored.ico" type="image/x-icon" />
 
-    <!-- CSS Imports -->
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link rel="stylesheet" href="assets/css/style.css" />
-    <title>Diciotech</title>
-  </head>
-  <body>
-    <header class="header">
-      <h1 class="header__title">
-        <img
-          src="/assets/img/brand/logo/colored.png"
-          width="480"
-          height="148"
-          alt="Diciotech"
-        />
-      </h1>
-      <h2 class="header__subtitle">
-        Um dicionário tech para pessoas que querem aprender mais sobre termos
-        técnicos dentro da tecnologia 📖
-      </h2>
-    </header>
-    <main class="main">
-      <div class="searchbar">
-        <svg
-          xmlns="http://www.w3.org/2000/svg"
-          class="searchbar__icon"
-          viewBox="0 0 16 16"
-        >
-          <path
-            d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"
-          />
-        </svg>
-        <input
-          id="search-input"
-          type="text"
-          name="search-input"
-          placeholder="Pesquisar..."
-          class="searchbar__input"
-        />
-      </div>
-      <section class="cards" id="cards"></section>
-    </main>
-    <script src="assets/js/script.js" type="module"></script>
-  </body>
-</html>
+  <!-- CSS Imports -->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="stylesheet" href="assets/css/style.css" />
+
+  <!-- Phosphor Icons -->
+  <script src="https://unpkg.com/@phosphor-icons/web"></script>
+
+  <title>Diciotech</title>
+</head>
+
+<body>
+  <header class="header">
+    <h1 class="header__title">
+      <img src="/assets/img/brand/logo/colored.png" width="480" height="148" alt="Diciotech" />
+    </h1>
+    <h2 class="header__subtitle">
+      Um dicionário tech para pessoas que querem aprender mais sobre termos
+      técnicos dentro da tecnologia 📖
+    </h2>
+
+    <button id="dark-mode-toggle" class="dark-mode-button">
```

Comment:
[sugestão] Acredito que assim conseguimos manter um padrão das outras nomenclaturas que já temos.

```suggestion
    <button id="dark-mode-toggle" class="darkmode__button">
```